user-manual/translate/translate-content.rst

```diff
@@ -41,11 +41,16 @@ work within it.
    typing into the data entry :term:`fields <field>`. Note that the content is
    shown in the source language above the data entry :term:`field`.
 
-.. TIP::
 
-   The tooltips (e.g., ISAD) will be shown in the new language because they
-   have been translated as part of the AtoM application using
-   `Transifex instructions <https://www.accesstomemory.org/community/translate/>`__.
+   .. TIP::
+
+   The tooltips (e.g., ISAD) may also be shown in the new language if they
```

Comment:
Hi @rpearcemoses we're going to handle this in a separate PR, but FYI: the content of the `TIP` tooltip here needs to be indented by 3 spaces, to match the start of the word TIP, like so: 

```
.. TIP::
   
   The tooltips (e.g., ISAD) may also be shown in the new language if they
   have been translated by our community of volunteer translators. For more 
   information, including how you can contribute translations included in 
   public releases, see the 
   `translation instructions <https://www.accesstomemory.org/community/translate/>`__ 
   on our wiki.
```

You can see that this threw an error in our build checker, here: https://travis-ci.org/artefactual/atom-docs/builds/571051870?utm_source=github_status&utm_medium=notification

Notice it gives you the line and the file where the issue is found. To avoid these errors before submitting, be sure to run `make clean html` locally before you add your files - this makes a build of the files locally in your `_build` directory that you can preview, and will also output any errors or warnings. We describe this further in the contribution docs on our wiki, here: 

* https://wiki.accesstomemory.org/Resources/Documentation/Contribute#Make_changes_and_commit_them_to_your_fork

In any case, these changes are not meant to be part of this current PR, so I will close it after cherry-picking the typo fix!